sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/WhereQuery.cs

```diff
@@ -94,8 +116,70 @@ public WhereQuery WhereStartsWith(string stringToCheck, string beginningString)
         /// <returns> ADT query that already contains SELECT and FROM. </returns>
         public WhereQuery WhereIsOfModel(string model, bool exact = false)
         {
-            Console.WriteLine(model);
-            Console.WriteLine(exact);
+            if (exact)
+            {
+                _clauses.Add(new WhereClause($"{QueryConstants.IsOfModel}('{model}', exact)"));
```

Comment:
small DRY violation here. Might be acceptable, but consider an additive model, like:

```csharp
string whereClause = $"{QueryConstants.IsOfModel}('{model}'";

if (exact)
{
    whereClause += ", exact";
}

whereClause += ")";
```

That may also beg for a `StringBuilder` for the number of concatenations.

Is readability better? Probably not.

Here's another approach:

```csharp
const string whereClauseFormat = $"{QueryConstants.IsOfModel}('{{0}}`{{1}})";

string useExact = exact ? ", exact" : "";

string.Format(
    whereClauseFormat,
    model,
    useExact);
```

It doesn't quite have the readability issue as string concatenation does.

In the end, everything is a trade-off and you have to decide which sin you want to live with the most.